weddineo/src/main/java/com/weddineo/firebase/service/FirebaseAppInitializer.java

```diff
@@ -6,25 +6,25 @@
 import com.weddineo.firebase.exception.WeddineoFirebaseRuntimeException;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
 import org.springframework.stereotype.Component;
 
 import javax.annotation.PostConstruct;
-import java.io.FileInputStream;
 import java.io.FileNotFoundException;
 import java.io.IOException;
 
 @Component
 @Slf4j
 public class FirebaseAppInitializer {
 
-    @Value("${firebase.private.key.path}")
-    private String firebasePrivateKeyPath;
+    @Value("classpath:firebase/weddineo-an-firebase-adminsdk.json")
```

Comment:
Why this path is not from properties file?

Reply:
While we are holding file in resources it is good practise to access them directly from the classpath, not by Java filesystem api. 
However, I am not sure if we should hold firebase-key in resources.